### ISSUE TITLE
Can pass file name instead of a input

### ISSUE THREAD

**Issue by darielopper** [Aivean/scala-to-java#6]
Its posible in future version pass a scala class filename as argument and generate a java class file as output???

**Comment by Aivean** [comment]
I don't think that new version will happen any time soon.  That being said, I think you can create your own script that pipes the source of your class into the tool and saves the result as a file.

I tried following and it worked:

     cat  EncodingUtil.scala | grep -Ev '^package' | java -jar ~/.scala-demystify.jar  > EncodingUtil.java

`grep -Ev '^package' `  was needed as compiler expects default package. Obviously, any non standard imports will not work (the tool tries to compile a single file, it's unaware of your dependencies).
